target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -153,6 +153,7 @@ case "$FIRMWARE" in
 	netgear,wndap360)
 		caldata_extract "art" 0x1000 0xeb8
 		;;
+	trendnet,tew-673gru|\
```

Comment:
Please maintain alphabetical order.

Reply:
Thanks for catching that (and apologies you needed to).